Tests/FluentAssertions.Specs/Primitives/DateTimeOffsetAssertionSpecs.cs

```diff
@@ -134,8 +134,8 @@ public class Be
         public void Should_succeed_when_asserting_datetimeoffset_value_is_equal_to_the_same_value()
         {
             // Arrange
-            DateTimeOffset dateTime = new DateTime(2016, 06, 04);
-            DateTimeOffset sameDateTime = new DateTime(2016, 06, 04);
+            DateTimeOffset dateTime = new DateTime(2016, 06, 04).ToDateTimeOffset();
```

Comment:
Would this be a place that the type is now so explicit, you could consider using `var`?

Reply:
Yes, normally I would.